packages/viewer/src/lib/types/depthBuffer.ts

```diff
@@ -102,7 +102,7 @@ export class DepthBuffer implements FrameImageLike {
       fallbackNormalizedDepth
     );
 
-    return depth * (far - near) + near / 2;
+    return depth * (far - near);
```

Comment:
NIT: if I'm understanding correctly, this probably ought to be `... + near` but `near==0` so it really doesn't matter mathematically but it might make more sense to a future reader.

Reply:
I've typically seen the orthographic cameras have a near/far of `[-1000, 1000]`, so this results in a depth between `[0, 2000]` in those cases (where if we added `... + near` we'd wind up with a depth between `[0, 1000]`. I did test with that addition, but saw incorrect results. Does the range from `[0, 2000]` in that case actually make sense in orthographic? Or is that potentially another symptom of some incorrect math somewhere else?

Reply:
Well, it does compute exactly the value the docstring claims it computes so maybe I should be quiet.  :shipit: